quarter-reports/main.py

```diff
@@ -0,0 +1,243 @@
+                        'last_commit': '![GitHub last commit](https://img.shields.io/github/last-commit/' + name_with_owner + '?display_timestamp=committer&label=%20)',
+                        'description': i['node']['description'],
+                        'createdAt': i['node']['createdAt'][0:10],
+                        'commits': '![GitHub total commit activity](https://img.shields.io/github/commit-activity/t/' + name_with_owner + '?label=%20)',
+                        'issues': '![GitHub Issues or Pull Requests](https://img.shields.io/github/issues/' + name_with_owner + '?label=%20)',
+                        'PRs': '![GitHub Issues or Pull Requests](https://img.shields.io/github/issues-pr/' + name_with_owner + '?label=%20)'})
+        else:
+          name_with_owner = i['node']['nameWithOwner']
+          old_repos.append({'name': '[' + i['node']['name'] + '](https://github.com/' + name_with_owner + ')',
+                        'last_commit': '![GitHub last commit](https://img.shields.io/github/last-commit/' + name_with_owner + '?display_timestamp=committer&label=%20)',
+                        'description': i['node']['description'],
+                        'createdAt': i['node']['createdAt'][0:10],
+                        'commits': '![GitHub total commit activity](https://img.shields.io/github/commit-activity/t/' + name_with_owner + '?label=%20)',
+                        'issues': '![GitHub Issues or Pull Requests](https://img.shields.io/github/issues/' + name_with_owner + '?label=%20)',
+                        'PRs': '![GitHub Issues or Pull Requests](https://img.shields.io/github/issues-pr/' + name_with_owner + '?label=%20)'})
+      elif i['node']['isArchived']:
+        name_with_owner = i['node']['nameWithOwner']
+        repos_archived.append({'name': '[' + i['node']['name'] + '](https://github.com/' + name_with_owner + ')',
+                      'last_commit': '![GitHub last commit](https://img.shields.io/github/last-commit/' + name_with_owner + '?display_timestamp=committer&label=%20)',
+                      'description': i['node']['description'],
+                      'createdAt': i['node']['createdAt'][0:10]})
+
+    has_next_page = results['data']['search']['pageInfo']['hasNextPage']
+    after_clause =', after:"{}"'.format(results['data']['search']['pageInfo']['endCursor'])
+
+  return new_repos, old_repos, repos_archived
+
+
+def generate_md(org, output_file, token, when):
+  """
+  Function to write on a markdown file all the data retrieved from a specific org by a given date.
+  """
+
+  new_repos, old_repos, archived_repos = query_github_repositories(org, token, when)
+  year = str(when)[0:4]
+  month = str(when)[5:7]
+  quarter = get_quarter(month)
+  total, archived, active = api_call_get_activity_data(org,token)
+  md = Template(f'''# {quarter} {year} Hyperledger Labs Update
+
+  ## Overall Health
```

Comment:
```suggestion
## Overall Health
```

Need to get rid of the leading spaces here since it includes these in the markdown file and the h2 heading is not recognized.

Reply:
Yep, my bad. Thanks!